src/shared/services/education-organizations-service.ts

```diff
@@ -5,7 +5,7 @@ import { Avo } from '@viaa/avo2-types';
 import { CustomError, getEnv } from '../helpers';
 
 export const fetchCities = async (): Promise<string[]> => {
-	let url: string | undefined = undefined;
+	let url: string | undefined;
```

Comment:
```
function bar(a: number?, b?: number) {
  a // => number | undefined | null
  b // => number | undefined
}
```

https://github.com/microsoft/TypeScript/issues/7426

Hier kunnen we een `?` gebruiken, en bij andere `| undefined` eigenlijk ook.